axelrod/strategies/gradualkiller.py

```diff
@@ -0,0 +1,37 @@
+from axelrod import Actions, Player, init_args
+C, D = Actions.C, Actions.D
+
+
+class GradualKiller(Player):
+    """
+    It begins by defecting in the first five moves, then cooperates two times.
+    It then defects all the time if the opponent has defected in move 6 and 7,
+    else cooperates all the time.
```

Comment:
Could you add a sentence here saying that this is designed to stop `Gradual` from winning? (And possibly why?)